src/Nox.Generator/Application/DomainEventHandlers/DomainEventHandler.template.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Generated
+
+#nullable enable
+
+using MediatR;
+using Nox.Application;
+using Nox.Messaging;
+
+using {{codeGeneratorState.DomainNameSpace}};
+using {{codeGeneratorState.ApplicationNameSpace}}.Dto;
+
+namespace {{codeGeneratorState.ApplicationNameSpace}}.DomainEventHandlers;
+
+internal abstract class {{className}}Base : INotificationHandler<{{entity.Name}}{{crudOperation}}>
+{
+    private readonly IOutboxRepository _outboxRepository;
+
+    protected {{className}}Base(IOutboxRepository outboxRepository)
+    {
+        _outboxRepository = outboxRepository;
+    }
+
+    public virtual async Task Handle({{entity.Name}}{{crudOperation}} domainEvent, CancellationToken cancellationToken)
+    {
+{{- if raiseIntegrationEvent }}
```

Comment:
the should be on the generator, and we would not generate a file / handler that is not needed

Reply:
Changes were applied.